src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -174,11 +173,11 @@ private void applyStringFilter(String filterString) {
             } else {
                 this.applyFilterExpression(Qualifier.EMPTY);
             }
-        } catch (ParseException ex) {
+        } catch (FilterException ex) {
             this.applyFilterExpression(Qualifier.EMPTY);
             // Overrides message in status bar
             UI.status.displayMessage("Panel " + (panelIndex + 1)
-                + ": Parse error in filter: " + ex.getMessage());
+                + ": " + ex.getMessage());
```

Comment:
This is a bit tangential, but maybe we could add the panel name here, defaulting to the index only if it's unnamed?